container-scan/action.yaml

```diff
@@ -120,32 +122,7 @@ runs:
         echo "</details>" >> pr-results.md
       shell: bash
 
-    - name: Check for previous report comment
```

Comment:
Why are you removing this?

Reply:
unintentionally deleted during removing/reorganizing the steps. will restore.